pkg/controller/directvolumemigrationprogress/validation_test.go

```diff
@@ -0,0 +1,99 @@
+package directvolumemigrationprogress
+
+import (
+	"testing"
+
+	migapi "github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
+	"github.com/konveyor/mig-controller/pkg/compat"
+	fakecompat "github.com/konveyor/mig-controller/pkg/compat/fake"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestReconcileDirectVolumeMigrationProgress_validateSpec(t *testing.T) {
```

Comment:
this test is added to make sure that pre MTC 1.4.3 CRs and new CRs both can co-exist without breaking the controller